var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -146,19 +146,23 @@ class Seacas(CMakePackage):
     variant("faodel", default=False, description="Enable Faodel")
     variant("matio", default=True, description="Compile with matio (MatLab) support")
     variant("metis", default=False, description="Compile with METIS and ParMETIS")
+    variant("pamgen", default=False, description="Compile with pamgen")
     variant("x11", default=True, description="Compile with X11")
-
+    variant("zlib", default=False, description="Compile with zlib")
     # ###################### Dependencies ##########################
     depends_on("cmake@3.22:", when="@2023-10-24:", type="build")
     depends_on("cmake@3.17:", when="@:2023-05-30", type="build")
     depends_on("mpi", when="+mpi")
-
+    depends_on("zlib", when="+zlib")
```

Comment:
Why `zlib` and not `zlib-api`?

Reply:
I was not aware of zlib-api.  It looks like it was created in August of 2023 and addresses the question I had asked a few years ago (https://groups.google.com/g/spack/c/ex-_auuyrVo/m/8ilGGingBAAJ?utm_medium=email&utm_source=footer&pli=1).  I will update to use zlib-api, unless you want to put up a pr...

Reply:
```suggestion
    depends_on("zlib-api", when="+zlib")
```

Reply:
I am testing this change currently before pushing up another commit.